kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ClustersResourceIntegrationTest.java

```diff
@@ -118,9 +129,7 @@ public void getCluster_ownCluster_returnsOwnCluster(String quorum) {
                         .setResourceName("crn:///kafka=" + clusterId)
                         .build())
                 .setClusterId(clusterId)
-                .setController(
-                    Resource.Relationship.create(
-                        baseUrl + "/v3/clusters/" + clusterId + "/brokers/" + controllerId))
+                .setController(relationship)
```

Comment:
same here, can we leave the old comment about the non-deterministic controller? Then, we can know that this is the only thing we can't check